src/form-actions/form-actions.js

```diff
@@ -0,0 +1,32 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('dd.ui.form-actions', [])
+        .directive('formActions', formActions);
```

Comment:
In this case we need to upgrade dd-ui and admin angular version to 1.5 and this needs some time.